src/common/snippets/src/lowered/pass/assign_registers.cpp

```diff
@@ -4,19 +4,32 @@
 
 #include "snippets/lowered/pass/assign_registers.hpp"
 
+#include "openvino/core/except.hpp"
+#include "openvino/core/type.hpp"
+#include "snippets/emitter.hpp"
 #include "snippets/itt.hpp"
+#include "snippets/lowered/expression.hpp"
+#include "snippets/lowered/expressions/buffer_expression.hpp"
 #include "snippets/lowered/linear_ir.hpp"
+#include "snippets/lowered/reg_manager.hpp"
+#include "snippets/op/fill.hpp"
+#include "snippets/op/horizon_max.hpp"
+#include "snippets/op/horizon_sum.hpp"
 #include "snippets/op/kernel.hpp"
-#include "snippets/snippets_isa.hpp"
-#include "snippets/utils/utils.hpp"
+#include "snippets/op/vector_buffer.hpp"
 
 // This header is needed to avoid MSVC warning "C2039: 'inserter': is not a member of 'std'"
```

Comment:
It looks like we don't need this comment anymore: we anyway need iterator header. Could you please remove it?

Reply:
Done